bluesky/plans.py

```diff
@@ -1413,6 +1413,50 @@ def fly(flyers, *, md=None):
     yield from bps.close_run()
 
 
+def x2x_scan(detectors, motor1, motor2, start, stop, num, *,
+             per_step=None, md=None):
+    """
+    Relatively scan over two motors in a 1:2 ratio
+
+    This is a pasteurized version of a theta2theta scan
```

Comment:
As much as I love metaphorical language, "generalized" might be friendlier here.

Reply:
I suspect that is me losing to the spell-checker....